frontend/src/utils/enums/housing.ts

```diff
@@ -20,6 +20,13 @@ export enum ContactPreference {
   EITHER = 'Either'
 }
 
+export enum EnquiryStatus {
+  NEW = 'New',
+  IN_PROGRESS = 'In Progress',
+  DELAYED = 'Delayed',
+  COMPLETED = 'Completed'
+}
+
```

Comment:
Duplicate of `ApplicationStatus`.

Reply:
removed and replaced status types in submission & enquiry with ApplicationStatus enum type as discussed